styles/styles.css

```diff
@@ -453,3 +457,126 @@ hr {
   border: 1px solid var(--white);
   cursor: pointer;
 }
+
+@media screen and (min-width: 768px) {
+  header img {
+    display: none;
+  }
+
+  header {
+    padding: 2rem 155px 0 155px;
+  }
+
+  header ul {
+    display: flex;
+  }
+
+  header .desktop-nav {
+    display: block;
+  }
+
+  header .desktop-nav li {
+    margin-left: 2rem;
+  }
+
+  /* Hero section */
+  #hero {
+    background-color: white;
+    padding: 0 1.5rem;
+    height: 700px;
+    border-radius: 0 0 0 5rem;
+    background-image: url("../images/desktop/desktop-bg.png");
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  #about-section,
+  #works,
+  #contact-form {
+    padding: 7.125rem 10rem;
+  }
+
+  .hero-content {
+    margin: 0 16.25rem;
+  }
+
+  .hero-content .name {
+    padding-top: 13rem;
+    margin-bottom: 0.75rem;
+    font-size: 3rem;
+    line-height: 3.75rem;
+  }
+
+  .desc {
+    margin-bottom: 0.75rem;
+    font-size: 1.25rem;
+    line-height: 1.75rem;
+  }
+
+  .desktop-menu {
+    font-size: 0.938rem;
+    font-weight: 500;
+    line-height: 20;
+    color: #344563;
+    text-align: center;
+  }
+
+  .card {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    height: 496px;
+    align-items: center;
+  }
```

Comment:
- [ ] Yet again another small detail seems to be forgotten here as you can see in the Figma design when the project card is hovered over with a mouse there's a border with a `#a7aeff` color and also a slight box shadow. so please make sure to implement those to provide a better UX. ✔
- Figma design:
  ![firefox_Axe18iHV4e](https://github.com/jecode93/portfolio/assets/104968236/83788636-59be-433b-8db6-2af57280d038)

Reply:
@AbelG101 You're right bro I didn't take care of the whole design.
I will do all the changes.

Thank you so much.